docs/src/pages/customization/palette/palette-ja.md

```diff
@@ -85,13 +85,16 @@ const theme = createMuiTheme({
     },
     // Used by `getContrastText()` to maximize the contrast between
     // the background and the text.
+    tonalOffset: 0.2,
+  },
+});
```

Comment:
buggy?

Reply:
Nice catch! I've deleted the errant translation